stable/insights-agent/README.md

```diff
@@ -79,6 +79,7 @@ Parameter | Description | Default
 `trivy.namespaceBlacklist` | Specifies which namespaces to not scan, takes an array of namespaces for example: `--set trivy.namespaceBlacklist="{kube-system,default}"` | nil
 `opa.role` | Specifies which ClusterRole to grant the OPA agent access to | view
 `opa.additionalAccess` | Specifies additional access to grant the OPA agent. This should contain an array of objects with each having an array of apiGroups, an array of resources, and an array of verbs. Just like a RoleBinding. | null
+`opa.installCRDs` | Specifies whether to install the `customcheckinstances.insights.fairwinds.com` CRD. If you are installing the `insights-agent` chart twice - i.e. once as `insights-agent` and once as `fw-telemetry` - you will want to set this flag to `false` on *one* of the installs, doesn't matter which. | true
```

Comment:
I don't think we need as much specificity here around fw-telemetry since this is a publicly consumable chart.